### ISSUE TITLE
Turkish translation Hoşgeldiniz

### ISSUE THREAD

**Issue by i0natan** [i0natan/nodebestpractices#139]
@grafiktr - Turkish translation can be officially kicked-off. You're the lead! I'll be at your disposal should you need anything. Please create a branch for this work and PR to master once it's 100% done. Once we load it, will put your name on top of the Turkish page

I would ask/recommend only 2 important guidelines (if something isn't clear - please let us know and will be glad to do it for you):

Keep the same content & formatting - the focus is on translation, should anyone want to modify the content or the graphics - let's PR first in English and them translate to Turkish
Duplicate pages - the content should be translated over a page duplication. Readme.MD became Readme.turkish.MD, all other files should be duplicated similarly. So the number of English & Turkish pages should be the same. Will create a bold link from the English home page to the Turkish home page (with bold credit of course...).

**Comment by grafiktr** [comment]
@ionatan - thank you for the support and the permission. the final version of the project will not let you down. 
Your highlights will be checked and rearranged as soon as possible, they will be borne in mind for further translations.

I have a request from you in order to keep the project easier and more regular. I would like to carry out the project on CrowdIn if you do not mind.

**Comment by i0natan** [comment]
@grafiktr sure, whatever platform makes your life easier.

You plan to work on this with other people or alone? if desirable, we can ask in Node.JS Turkey FB group for contributors

**Comment by grafiktr** [comment]
@i0natan thank you for your understanding. would you like me to transfer the project to CrowdIn or can you take care of it?
I prefer to work alone when it comes to Turkish but for other languages, of course I will need help from other contributors. But as I said, I can handle the Turkish part by myself.

**Comment by i0natan** [comment]
@grafiktr I think we can start with Turkish first and then when it's ready move to others. What does it mean technically to transfer a project to CrowdIn? will appreciate few words about it works

**Comment by grafiktr** [comment]
@i0natan https://support.crowdin.com/github-integration/ here is the detailed answer to your question

**Comment by grafiktr** [comment]
Okay, just let me know when it is done. Thank you!

**Comment by i0natan** [comment]
@grafiktr anything you need from my part before starting?

**Comment by grafiktr** [comment]
@i0natan  That’s it for now, thank you for everything. If you just upload the project as soon as possible to crowdin, I can start the translation immediately. And one thing to ask, would you mind if you do not let anyone to participate as a proofreader to this project? I prefer to work alone that’s why

**Comment by i0natan** [comment]
You lead... whatever is more convenient to you

**Comment by grafiktr** [comment]
@i0natan Is there any progression about transferring the project to CrowdIn?

**Comment by i0natan** [comment]
@grafiktr really sorry, missed your message somehow. I think that we had some miscommunication - I understood that you are going to use Crowding, do you u need me tell perform anything?

**Comment by grafiktr** [comment]
@i0natan no problem, i just do not want to postpone the work. only thing i need from you is that to download the project on crowdin

**Comment by i0natan** [comment]
@grafiktr OK, I thought that you're doing this. LMK check how to perform this

**Comment by behcet** [comment]
@i0natan "أهلا بك" has nothing to do with Turkish

See: [Turkish alphabet](https://en.wikipedia.org/wiki/Turkish_alphabet)

**Comment by i0natan** [comment]
@behcet Can you then help me translate "Welcome" or "Good luck" to Turkish?

**Comment by behcet** [comment]
Sure thing. 

Welcome = Hoşgeldiniz

Good luck = Başarılar

**Comment by puskuruk** [comment]
Ping.

**Comment by GokhanMaden** [comment]
Hey @puskuruk , I saw your message two days ago but I waited for Yoni's answer. I think nobody interested Turkish translation. On the other hand, we can do it. I will send an email tomorrow morning, can we continue talking from there?

And @i0natan , still waiting for your answer. It's nice to find out you're interested Turkish translation.

**Comment by puskuruk** [comment]
OK, I'm waiting your email. @GokhanMaden

**Comment by i0natan** [comment]
@GokhanMaden Welcome aboard 👍 👍 

We'd love to get your help.

" I think nobody interested Turkish translation"

Oh no. We absolutely will appreciate a translation for a language that is spoken by many millions. I was on a vacation. 

Let me know how can I help in putting this in crowdin?

**Comment by i0natan** [comment]
@BrunoScheufler Just noticed that you are the sultan of the Turkish translation, assigning to you

**Comment by BrunoScheufler** [comment]
@GokhanMaden @puskuruk @behcet I'm excited for your contributions to this project! If you need anything or finish a section, don't hesitate to mention me, I'll handle the guideline reviews, etc 👍

**Comment by rozehan** [comment]
Hello,
I am interested in this project. This week is the right time :)

**Comment by puskuruk** [comment]
@rozehan we need to share tasks about this translation. Do you have an idea for how we track this project? Because we looked crowdin with @GokhanMaden and this was not free.

**Comment by BrunoScheufler** [comment]
@puskuruk if you need anything I can help with, don't hesitate to notify me. You could use some kind of board like trello to manage tasks if you need that, otherwise you could share a list in here 🤔 Guess there's a lot of ways to achieve this. Also, it would make sense to find a solution to contributing, since we will open a branch  for your translation, but you'd still have to open PRs to that to add your content. Tools like crowdin would be awesome for sure, but costly..

**Comment by rozehan** [comment]
@puskuruk Crowdin have a request form for open source projects. I don't know if it's a ridiculous question, but can't it be tried?
https://crowdin.com/page/open-source-project-setup-request

**Comment by mperk** [comment]
Hi, I started for the translation turkish on my local. 
Can you give me permission Please?

**Comment by mperk** [comment]
to @i0natan @BrunoScheufler

**Comment by BrunoScheufler** [comment]
@mperk sorry but what do you mean by giving permission? Once you've made progress in translation that you want to save, you could simply open a PR to the branch `turkish-translation`, where we will collect all translation progress and merge into master at one point 👍

**Comment by mperk** [comment]
Ok thank you @BrunoScheufler 
I got it wrong the first message. After I forked, I continue to translate on my turkish-translation branch.

**Comment by mperk** [comment]
Hi @BrunoScheufler,
Can you create a turkish-translation branch? I want to merge the branch 
[https://github.com/mperk/nodebestpractices/tree/turkish-translation](url)
Because, maybe someone want to help me

**Comment by BrunoScheufler** [comment]
@mperk branch is created, ready for the PR 😄

**Comment by i0natan** [comment]
👍 👍

**Comment by murataksu740** [comment]
I want to translate it to Turkish. What should I do?

**Comment by alicanyildirim** [comment]
Hi everyone, I have seen that the translation to Turkish hasn't been completed since early 2018. Do you guys need any help, or is this translation scrapped?

**Comment by goldbergyoni** [comment]
@BrunoScheufler Any thoughts on this? Seems to me like we could greatly benefit from having a Turkish translation

**Comment by batudmr** [comment]
Hello everyone, if you still need Turkish translation, I have forked the project and I can start translating right away. Is there anything I should be aware while translating the content? Any suggestions or requirements?

**Comment by LeaveNhA** [comment]
If anyone wants to lead, I can contribute. Also, if I can find time to take the whole issue, I can make it alone. I need time to evaluate my calendar. I'm gonna inform you here in 20 days.

**Comment by TarikSogukpinar** [comment]
Is there someone who manages the Turkish translation process? I would like to help with the document

**Comment by MutanPlex** [comment]
Translation using Google Translate has resulted in a completely unrelated translation to the software context.

**Comment by MutanPlex** [comment]
Translation using Google Translate has resulted in a completely unrelated translation to the software context. Using my 11 years of software experience, I can provide translations in Turkish that are completely related to software. If the Turkish translation is still ongoing, @goldbergyoni.
